flagscale/train/hetero/train_llama.py

```diff
@@ -0,0 +1,292 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Do not copy a `train_llama.py`, which should be unified training entrance. You can put the functional changes that must be rewritten in hetero folder.

Reply:
Great idea! This will definitely be improved in the next pull request.